apps/dh/api-dh/source/DataHub.WebApi.Tests/TestServices/GraphQLTestService.cs

```diff
@@ -13,26 +13,36 @@
 // limitations under the License.
 
 using System;
+using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.Databricks;
 using Energinet.DataHub.Edi.B2CWebApp.Clients.v1;
 using Energinet.DataHub.Edi.B2CWebApp.Clients.v3;
+using Energinet.DataHub.MarketParticipant.Authorization.Model;
+using Energinet.DataHub.MarketParticipant.Authorization.Services;
+using Energinet.DataHub.Measurements.Abstractions.Api.Queries;
 using Energinet.DataHub.Measurements.Client;
+using Energinet.DataHub.Measurements.Client.Authorization;
+using Energinet.DataHub.Measurements.Client.ResponseParsers;
```

Comment:
Renamed to `Energinet.DataHub.Measurements.Client.Mappers`

Reply:
Fixed